### PR TITLE
chore: Add pull request template

### PR DESCRIPTION
**Pull request by nlauchande**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [X] Chore
- [ ] Documentation Update

## Description
A PR description would help on consistency and guide better documentation in the project.

## Related Tickets & Documents
N/A

## Added/updated tests?
- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by neubig** [comment]
Thanks for the contribution @nlauchande !

I'm wondering what the other maintainers think about PR templates -- I think some structure is good, but too much structure adds extra red tape that takes time.

Personally I think that something like this but a little simpler might be good. For instance:

```markdown
<!-- please describe the PR so that others reviewing understand the motivation -->

<!-- if this fixes an issue, please write "Fixes [issue_url]" so the issue is auto-closed -->

<!-- please check if you added unit tests for new code, and add them if not -->
```

**Comment by nlauchande** [comment]
Cool . Will update with your suggestion .

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (amanape) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: I don't think there is an urgent requirement to need a template for PRs yet. We already have basic guidelines described [here](https://github.com/OpenDevin/OpenDevin/blob/main/CONTRIBUTING.md#1-pull-request-title) that are good enough for this stage

CC: @rbren

**Comment by neubig** [comment]
Hi @nlauchande , first thanks a bunch for the contribution! We had a discussion among the maintainers and I think we're not going to add a PR template at the moment just to minimize the amount of overhead that goes into making a PR. We may revisit this decision later, but we'll close this PR for now. We'd definitely welcome other contributions though!

**Pull request closed without merging**